src/agent/rustjail/src/container.rs

```diff
@@ -473,11 +473,11 @@ fn do_init_child(cwfd: RawFd) -> Result<()> {
         if let Err(e) = sched::setns(fd, s) {
             if s == CloneFlags::CLONE_NEWUSER {
                 if e.as_errno().unwrap() != Errno::EINVAL {
-                    write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str());
+                    write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str())?;
```

Comment:
It's better not change the logic in a warning fix commit unless you have done enough testing
what about use `write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str()).ok()`

Reply:
Why do you feel it is safe to ignore the write error here?

Reply:
@c3d there is not much we can do with this error.